index.js

```diff
@@ -130,7 +130,7 @@ function parseRubyEngineAndVersion(rubyVersion) {
   } else if (rubyVersion === '.tool-versions') { // Read from .tool-versions
     const toolVersions = fs.readFileSync('.tool-versions', 'utf8').trim()
     const rubyLine = toolVersions.split(/\r?\n/).filter(e => /^ruby\s/.test(e))[0]
-    rubyVersion = rubyLine.split(/\s/)[1]
+    rubyVersion = rubyLine.split(/\s+/)[1]
```

Comment:
It makes me think we should really do https://github.com/ruby/setup-ruby/pull/717/files#r1982040471 but I merged this as-sis to fix it ASAP.
We should also add a test for .tool-versions

Reply:
30755d8bf4ebfa38caa5ad3a6b808846393196a5